SeaPublicWebsite/Models/EnergyEfficiency/QuestionOptions/Epc.cs

```diff
@@ -20,15 +21,63 @@ public class Epc
         public FloorConstruction? FloorConstruction { get; set; }
         public FloorInsulated? FloorInsulated { get; set; }
         public HomeAge? ConstructionAgeBand { get; set; }
+        public RoofConstruction? RoofConstruction { get; set; }
+        public RoofInsulated? RoofInsulated { get; set; }
+        public GlazingType? GlazingType { get; set; }
+    }
 
+    public class EpcInformation
+    {
+        public string EpcId { get; set; }
+        public string Address1 { get; set; }
+        public string Address2 { get; set; }
+        public string Postcode { get; set; }
 
-        public int? GetHouseNumber() {
-            var houseNumberFromFirstLine = Epc.GetIntegerFromStartOfString(this.Address1);
-            var houseNumberFromSecondLine = Epc.GetIntegerFromStartOfString(this.Address2);
+        public EpcInformation FixFormatting()
```

Comment:
I think this would be better in a constructor for this class. Then you can't forge to call it.